config.yml

```diff
@@ -57,5 +57,23 @@ default:
       "iShares MSCI ACWI UCITS ETF <USD (Accumulating)>" =
       "https://www.ishares.com/uk/individual/en/products/251850/ishares-msci-acwi-ucits-etf/"
     )
-  
 
+2023Q4:
+  pacta_financial_timestamp: "2023Q4"
+  ishares_date: "20231231"
+  bonds_indices_urls: !expr |- 
+    c(
+      "iShares Global Corp Bond UCITS ETF <USD (Distributing)>" = 
+      "https://www.ishares.com/uk/individual/en/products/251813/ishares-global-corporate-bond-ucits-etf/"
+    )
+  equity_indices_urls: !expr |-
+    c(
```

Comment:
After the fact NIT: This makes sense to me for now, but out of curiosity, why are we escaping R vector creation, when yaml supports vectors?

This would import the object as a list instead of a vector, but that should be coercible to a normal named vector pretty easily

Reply:
Honestly, I don't remember why we started doing this lol. Happy if you want to open an issue to look into a better approach